src/tests/NewCard.test.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { fireEvent, render, screen } from '@testing-library/react'
+import { CardSide, NewCard } from '../components/AppMain/NewCard/NewCard'
+import { Buttons } from '../components/AppMain/NewCard/Buttons/Buttons'
+import { AppHeader } from '../components/AppHeader'
+
+test('error alert when front or back side is empty', () => {
+	const onNextMock = jest.fn()
+	const onSaveMock = jest.fn()
+    render(<AppHeader/>)
+	render(<NewCard />)
+	render(
```

Comment:
**issue:** You should not render these components separately as you don't test integration this way. You should render whole parent that has these components included